server/db/schema.sql

```diff
@@ -12,6 +12,7 @@ CREATE TABLE products (
   default_price TEXT
 );
 
+CREATE INDEX idx_product_id ON products(id);
```

Comment:
good idea to add this here for when you need to rerun the db schema file - I'll do that too! I wonder if you could just define the index in the CREATE TABLE itself? For mysql you can add a line at the end that says INDEX index_name (column_name) just like you would with primary key but I couldn't find a postgres specific example.

Reply:
hmm i'm not sure. i just saw in the postgre docs that they used create index command, so that's what i did